dev-python/raven/raven-6.10.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 python3_8 )
```

Comment:
Hmm, I wonder if dropping python3.6 will make it harder to update revdeps etc. Just include it here until it's dropped tree-wide.